internal/storage/storage.go

```diff
@@ -0,0 +1,141 @@
+package storage
+
+import (
+	"database/sql"
+)
+
+const (
+	ParcelStatusRegistered = "registered"
+	ParcelStatusSent       = "sent"
+	ParcelStatusDelivered  = "delivered"
+)
+
+type Parcel struct {
+	Number    int64
+	Client    int
+	Status    string
+	Address   string
+	CreatedAt string
+}
+
+type ParcelStore struct {
+	db *sql.DB
+}
+
+func NewParcelStore(db *sql.DB) ParcelStore {
+	return ParcelStore{db: db}
+}
+
+func (s *ParcelStore) Add(p Parcel) (int64, error) {
+	res, err := s.db.Exec("INSERT INTO parcel (client,status,address,created_at) VALUES (:c, :s, :a, :cr)",
```

Comment:
Если нет желания полноценно называть переменные, то можно использовать неименованные параметры
```suggestion
	res, err := s.db.Exec("INSERT INTO parcel (client,status,address,created_at) VALUES (?, ?, ?, ?)", p.Client, p.Status, p.Address, p.CreatedAt)
```